pom.xml

```diff
@@ -194,9 +194,11 @@
       </activation>
       <build>
         <plugins>
+          <!--
```

Comment:
I don't think that this should have ended up in this PR. You basically _disabled_ the integration tests.

To be honest, I don't  think that _any_ of these changes to `pom.xml` are actually relevant or plausible in the context of resolving #96.

Only bumping `mojo-parent` from 40 to the latest version (50) seems like a good idea in general, but that should be done in a separate issue/PR. FYI: @khmarbaise

Reply:
PS: Looking at all these commits, it seems you mistakenly pushed too many of them (from some private/repo-specific branch).

Reply:
Remove the mistakenly pushed commits with pom changes.